requirements_dev.txt

```diff
@@ -18,6 +18,7 @@ h5py
 scikit-image
 pandas
 rasterio
+gdal
```

Comment:
same as above, I would not include it in here...

Reply:
You are right. I'll remove those.